model/pokemon.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const uuidv1 = require('node-uuid').v1;
+
+module.exports = function(opts){
+  this.name = opts.name;
+  this.type = opts.type;
+  this.pokedexNUM = opts.pokedexNUM;
+  this.moves = opts.move;
```

Comment:
I saw that ``this.moves`` is more than one thing, so when we move on to more complex models using MongoDB, you should consider making properties with multiple values into an array or a nested object.